sandbox/src/GameWorld.h

```diff
@@ -45,11 +45,11 @@ class GameWorld: public Engine::Layer
 GameWorld::GameWorld(Engine::Ref<Engine::VertexBuffer> vb, Engine::Ref<Engine::IndexBuffer> ib, const std::string &name)
 : Layer(name)
 {
-  m_Mesh = Engine::Mesh::Create(vb, ib, "sandbox/assets/shaders/World.glsl");
+  m_Mesh = Engine::Mesh::Create(vb, ib, "sandbox\\assets\\shaders\\World.glsl");
```

Comment:
was für eine lösung gibt es da in c++17?

Reply:
angeblich soll https://en.cppreference.com/w/cpp/header/filesystem dafür implementiert worden sein, habs aber noch nicht getestet